<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>594</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>11</y>
      <width>941</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QTextBrowser" name="text_browser">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;aaa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="cursorWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_edit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConsole">
    <property name="title">
     <string>Console</string>
    </property>
    <widget class="QMenu" name="menuChanger_l_affichage_2">
     <property name="title">
      <string>Changer l'affichage</string>
     </property>
     <addaction name="action_raw_display"/>
     <addaction name="action_precompiled_display"/>
    </widget>
    <addaction name="action_clear"/>
    <addaction name="menuChanger_l_affichage_2"/>
   </widget>
   <addaction name="menuConsole"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_load_plugin">
   <property name="text">
    <string>Charger un plugin</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>Vider</string>
   </property>
  </action>
  <action name="actionbrute">
   <property name="text">
    <string>Brute</string>
   </property>
  </action>
  <action name="actionPr_compil">
   <property name="text">
    <string>Précompilé</string>
   </property>
  </action>
  <action name="action_raw_display">
   <property name="text">
    <string>Brut</string>
   </property>
  </action>
  <action name="action_precompiled_display">
   <property name="text">
    <string>Précompilé</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
